pom.xml

```diff
@@ -31,22 +31,22 @@
         <maven.build.timestamp.format>yyyyMMdd</maven.build.timestamp.format>
         <timestamp>${maven.build.timestamp}</timestamp>
 
-        <hazelcast.version>4.1</hazelcast.version>
+        <hazelcast.version>5.0.5</hazelcast.version>
```

Comment:
I would rather see a latest here (5.4.0), otherwise CVEs will show up on the mvnrepository site.

Reply:
Since we use a provided dependency, we should not see them. 🤔

Reply:
Provided dependencies are not pulled in when you add `hazelcast-wm` as a dependency, but they are still reported (along with test dependencies) on that website, and people (our customers) don't understand that and report it like they are real vulnerabilities. That's exactly what happened with 4.0, see https://mvnrepository.com/artifact/com.hazelcast/hazelcast-wm/4.0

Reply:
Didn't know that, ty